web/packages/teleport/package.json

```diff
@@ -20,7 +20,7 @@
     "@gravitational/design": "1.0.0",
     "@gravitational/shared": "1.0.0",
     "xterm": "^5.0.0",
-    "xterm-addon-fit": "^0.5.0"
+    "xterm-addon-fit": "^0.7.0"
```

Comment:
can you remove the `^` plz

Reply:
If there's good reason, sure. But we use this in at least 3 other places, so what's the justification for such a change?

Reply:
Yeah I'm hoping that we can remove all the carets throughout our dependency files as this doesn't actually pin the version.

This assumes that the author of the package will not make breaking changes in the minor/patch releases and also opens up the possibility two people will have different versions of a dependency....in this case`0.7.0` and another `0.7.1`. This becomes extra problematic when it's on a major version like `xterm` above as one person could have `5.0.0` and another `5.1.0`.

> Allows changes that do not modify the left-most non-zero digit in the [major, minor, patch] tuple. In other words, this allows patch and minor updates for versions 1.0.0 and above, patch updates for versions 0.X >=0.1.0, and no updates for versions 0.0.X.

https://docs.npmjs.com/cli/v6/using-npm/semver#caret-ranges-123-025-004

Reply:
@hatched is this a common enough issue to warrant such a stringent policy? In an ideal, semantically versioned universe the discrepancy between 5.0.0 and 5.1.0 shouldn't matter, and we're getting improvements in higher versions.

Reply:
While this may (or may not) be a good change to make, the purpose of this particular PR is to fix the SSH connection issue. If we want to update our practices for how we handle dependencies, we should dedicate a separate discussion to that rather than piggy back on a separate fix.

Reply:
@ibeckermayer that's the kicker though isn't it 😉 

My goal is just that the builds are reproducible, but yeah this can 100% land as-is 😁